pom.xml

```diff
@@ -16,7 +16,7 @@
     <dependency>
       <groupId>org.apache.struts</groupId>
       <artifactId>struts2-core</artifactId>
-      <version>2.3.3</version>
+      <version>2.3.35</version>
```

Comment:
Does this svg image work?

[![DepShield Badge](https://jcava0.dev.depshield.sonatype.org/badges/bigspotteddog/test-app12/depshield.svg)](https://depshield.github.io)